stellarphot/plotting/aij_plots.py

```diff
@@ -32,11 +34,8 @@ def seeing_plot(raw_radius, raw_counts, binned_radius, binned_counts, HWHM,
     file_name : optional, string
         if entered, file will save as png with this name
 
-    gap : number
-        the distance between the aperture and the inner annulus
-
-    annulus_width : number
-        the distance between the inner and outer annulus
+    aperture_settings : optional, `ApertureSettings`
+        The aperture settings used to create the plot.
```

Comment:
You should document the choices made if `aperture_settings` is not provided.

Reply:
Done in the latest revision